library/Icinga/Application/Config.php

```diff
@@ -412,17 +412,16 @@ public static function app($configname = 'config', $fromDisk = false)
      */
     public static function module($modulename, $configname = 'config', $fromDisk = false)
     {
-        if (! isset(self::$modules[$modulename])) {
-            self::$modules[$modulename] = array();
+        if (! isset(static::$modules[$modulename])) {
+            static::$modules[$modulename] = array();
```

Comment:
@lazyfrosch: when working with static properties, I'd suggest to use self::$property instead of static::$property. Motivation: there are late static method calls, but no late static properties. So, with B and C inheriting from A, B::$property equals C::$property, with $property defined in A - no matter whether you are going to use self or static. So, while replacing self with static has no effect in this commit, I'd opt for keeping self - as it doesn't suggest people not so deep into PHP a reality that doesn't exist. Namely, late static properties. At least not unless you break their reference in the child class.